zebra/zebra_routemap.c

```diff
@@ -576,7 +578,7 @@ DEFUN (zebra_route_map_timer,
        ZEBRA_STR
        "Set route-map parameters\n"
        "Time to wait before route-map updates are processed\n"
-       "0 means event-driven updates are disabled\n")
+       "0 means route-map changes are run immediately instead of delaying\n")
```

Comment:
hmm - the code in places like `zebra_route_map_mark_update()` sure looks like it will not schedule the timer if the timeout is zero? it's a little confused, because setting the value to zero triggers _one_ immediate run _if_ a timer was previously scheduled.

Reply:
I believe setting the delay-timer to 0 and not -running any route-map processing is incredibly... BAD.  I'm changing the behavior to 0 means route-map changes are run immediately